pandas/core/util/numba_.py

```diff
@@ -129,94 +127,3 @@ def impl(data, *_args):
             return impl
 
     return numba_func
-
-
-def split_for_numba(arg: FrameOrSeries) -> Tuple[np.ndarray, np.ndarray]:
```

Comment:
are these not used in the window functions?

Reply:
Nope. Only the groupby functions so that's why I moved them to the `groupby/numba_.py` file